app/helpers/administrate/application_helper.rb

```diff
@@ -52,7 +52,7 @@ def resource_index_route(resource_name)
     end
 
     def sanitized_order_params(page, current_field_name)
-      collection_names = page.item_includes + [current_field_name]
+      collection_names = page.item_associations + [current_field_name]
```

Comment:
If I have this right, it looks like `item_includes` won't be used any more, so I think it can be removed. Perhaps not straightaway, but starting with a deprecation warning like we do at https://github.com/thoughtbot/administrate/blob/fd83fbbb125446f782d9713229c69dcc1d826843/lib/administrate/field/associative.rb#L40-L42.